.github/azure-workflows/bootstrap.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-mdutil -sa
```

Comment:
I intended to use mdutil to disable Spotlight indexing. On Travis CI, it's disabled by default, but IIRC, on Azure it's not.

Reply:
Replaced with `sudo mdutil -a -i off`.